pkg/doc/did/doc_test.go

```diff
@@ -10,6 +10,7 @@ import (
 	"encoding/hex"
 	"encoding/json"
 	"encoding/pem"
+	"github.com/magiconair/properties/assert"
```

Comment:
Can we use "require.Equals" as used in others rather than importing a new lib ? ref - https://github.com/hyperledger/aries-framework-go/blob/master/pkg/doc/did/doc_test.go#L82

Reply:
sure